packages/legend-transac/src/@types/event/events.ts

```diff
@@ -199,6 +235,7 @@ export const microserviceEvent = {
     'COINS.UPDATE_SUBSCRIPTION': 'coins.update_subscription',
     'LEGEND_MISSIONS.COMPLETED_MISSION_REWARD': 'legend_missions.completed_mission_reward',
     'LEGEND_MISSIONS.ONGOING_MISSION': 'legend_missions.ongoing_mission',
+    'LEGEND_RANKINGS.SEND_EMAIL_TO_WINNERS': 'legend_rankings.send_email_to_winners',
```

Comment:
In contrast to a saga command, which explicitly dictates the order of actions to a specific microservice, an event simply represents something that has occurred within a microservice, and subscribers react based on their interest. The event doesn't issue directives or control what actions should follow. Therefore, a more fitting name for the event would be 'LEGEND_RANKINGS.WINNERS_DETERMINED' or something similar.

Reply:
Based on that definition, what is happening is that the rankings have finished, which is why I think the event name `legend_rankings.rankings_finished` is better